.rubocop.yml

```diff
@@ -0,0 +1,197 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.3
+  Exclude:
+    - db/schema.rb
+    - vendor/**/*
+    - gemfiles/vendor/**/*
+    - tmp/**/*
+
+LineLength:
```

Comment:
In my .rubocop.yml files, all cops are namespaced and listed alphabetically. Can we do the same here?